### ISSUE TITLE
Removing a sentence in a document changes predicted entities throughout document

### ISSUE THREAD

**Issue by erotavlas** [explosion/spaCy#4873]
I'm encountering an issue where when I load my latest NER model and test it on a document, it produces a different result if I remove a single sentence from the beginning of the document

I tested with two separate models and you can see an example below

I added a sentence to the first document with a fictitious person entity

![1_1](https://user-images.githubusercontent.com/10454265/71741379-c2365200-2e2c-11ea-84cd-33cea2e8ff8b.png)

Then removed the first sentence

![2_1](https://user-images.githubusercontent.com/10454265/71741388-ca8e8d00-2e2c-11ea-984d-194baec2f69f.PNG)

Is this normal?

I find it strange that such a short sentence would have such a profound effect on the predictions

Models were trained using spacy 2.1.8 
with custom vocabulary using word2vec using the following parameters

```
from gensim.models import Word2Vec
word2vec = Word2Vec(all_words, min_count=2, window=5, size=200, sg=0, workers=5)
```

**Comment by adrianeboyd** [comment]
Hmm, it does seem strange, but it's hard to know for sure without being able to replicate it on our end.

Does your pipeline contain more than one NER component: `EntityRuler` or `EntityRecognizer` or really anything that some entities before this NER model gets run? It reminds me a bit of #4232 or maybe even #4330, which aren't fixed until v2.2.

Or possibly something related to components with conflicting word vectors related to your issue in #4872?

Can you provide more information about the whole pipeline and how the word vectors were initialized for the custom spacy model?

It would also be useful to know if you run into the same problems with v2.2.3?

**Comment by erotavlas** [comment]
I retrained the model on version 2,2,3 and I no longer experience the issue. Must have been a bug with the previous version, but I haven't done any extensive testing to verify.  If I encounter it again I'll reopen the issue, but so far looks good.
